core/driver/sources/src/main/resources/reference.conf

```diff
@@ -0,0 +1,81 @@
+# Default configuration for the org.virtuslab.ideprobe.config.IdeProbeConfig
+probe {
+
+  # intellij is for org.virtuslab.ideprobe.config.IntellijConfig - default values below
+  intellij {
+    plugins = []
+    version {
+      build = "212.5080.55"
+      release = "2021.2.1"
+    }
+    # path = "" // use this config with proper path string if you want to use non-default IntellijConfig
+  }
+
+  # workspace is for Option[org.virtuslab.ideprobe.config.WorkspaceConfig] - default None - comments added as a reference
+  # workspace {
+  #   path { // for org.virtuslab.ideprobe.dependencies.Resource
+  #     uri {
+  #       str = ""
+  #     }
+  #     path {} // for java.nio.file.Path interface - choose proper implementation and add proper parameter names
+  #     cause {
+  #       s = ""
+  #     }
+  #   }
+  #   existing {} // for java.nio.file.Path interface - choose proper implementation and add proper parameter names
+  #   branch = ""
+  #   tag = ""
+  #   commit = ""
+  # }
+
+  # resolvers id for org.virtuslab.ideprobe.config.DependenciesConfig.Resolvers - default values below
+  resolvers {
+    intellij {
+      repositories = []
```

Comment:
Hmm is the default empty here? 🤔 aren't there some default JetBrains-provided repos set as defaults here?

Reply:
Look at https://github.com/VirtusLab/ide-probe/blob/master/core/driver/sources/src/main/scala/org/virtuslab/ideprobe/config/IdeProbeConfig.scala - what are the default values in scala code. I just used the same for the reference.conf

Reply:
Wow okay, probably the defaults are then resolved in some opaque way (i.e.  _if `repositories` empty, then assume something_)

Reply:
It might be... To be honest, I just followed the issue's title literally: "Create reference.conf to keep default values for IdeProbeConfig" :)

Reply:
Sure, that's not a thing to be rectified on this PR... pls open a separate issue to diagnose & possibly fix that :)

Reply:
it should be done with https://github.com/VirtusLab/ide-probe/issues/253